xtext-website/_posts/releasenotes/2018-05-23-version-2-14-0.md

```diff
@@ -19,6 +19,7 @@ Xtext and Xtend 2.14 are able to compile with and against Java 9 & 10. Here are
 - Java 9 requires Tycho 1.1 or newer, Java 10 requires Tycho 1.2
 - Java 9 requires Oxygen.1a at least, Java 10 requires Oxygen.3a. Eclipse Photon works fine for both.
 - Java 10 requires the org.objectweb.asm library in version 6.1.1
+- Tycho 1.1 adds Java 9 compatibility, but Java 10 support is still under development and provided with 1.2.0-SNAPSHOT so far. If you are using Java 10 already together with Xtext 2.14 and like to build with Tycho you might need to modify your POM files to use the snapshot until Tycho 1.2.0 is final.
```

Comment:
>  you might need to modify your POM files 
you have to modify the parent's project pom.xml and and change `tycho-version` property...